scripts/ci-test-lane.sh

```
@@ -0,0 +1,384 @@
+    cstatus=0
+    echo "::group::isolation: $cls (budget "${cls_timeout}"s)"
+    xcodebuild_test "$cls_timeout" "$LOG_DIR/iso-$cls.log" "$RESULT_DIR/iso-$cls.xcresult" 1 \
+      "-only-testing:$TARGET/$cls" || cstatus=$?
+    echo "::endgroup::"
+    cls_secs=$(( $(date +%s) - cls_start ))
+    if [ "$cstatus" -eq 0 ]; then
+      echo "$cls|pass|$cls_secs" >> "$ISOLATION_LINES"
+    elif [ "$cstatus" -eq 124 ]; then
+      echo "$cls|timeout|$cls_secs" >> "$ISOLATION_LINES"
+      if [ -z "$HUNG_CLASS" ]; then HUNG_CLASS="$cls"; fi
+      echo "::error::isolation: $cls HUNG (exceeded its "${cls_timeout}"s class budget)"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)